machine/accton/accton_as4610_54/machine.make

```diff
@@ -0,0 +1,38 @@
+# Makefile fragment for Accton AS4610_54
+
+ONIE_ARCH ?= armv7a
+
+VENDOR_REV ?= 0
+
+# Translate hardware revision to ONIE hardware revision
+ifeq ($(VENDOR_REV),0)
+  MACHINE_REV = 0
+else
+  $(warning Unknown VENDOR_REV '$(VENDOR_REV)' for MACHINE '$(MACHINE)')
+  $(error Unknown VENDOR_REV)
+endif
+
+UBOOT_MACHINE = AS4610_54
```

Comment:
Generic comment : If we do not specify UBOOT_MACHINE in machine.make, would uboot
infra is not build.
e.g there could be platforms, which might use other uboots. i.e. not the one build by onie.